lib/ASTGen/Sources/ASTGen/Exprs.swift

```diff
@@ -233,7 +233,7 @@ extension ASTGenVisitor {
     if let node = node.capture {
       result.bracketRange = self.generateSourceRange(node)
       let captures = node.items.lazy.map { node in
-        self.generate(closureCapture: node)
+        self.generate(closureCapture: node, capturedSelfDecl: &result.capturedSelfDecl)
```

Comment:
Using an `inout` argument in a lazy transform feels like it has the potential to be a footgun, though I guess it doesn't really matter in practice given we know `bridgedArray` iterates in order